docs/streams/upgrade-guide.html

```diff
@@ -191,6 +191,17 @@ <h3><a id="streams_api_changes_210" href="#streams_api_changes_210">Streams API
         details, see <a href="https://cwiki.apache.org/confluence/display/KAFKA/KIP-312%3A+Add+Overloaded+StreamsBuilder+Build+Method+to+Accept+java.util.Properties">KIP-312</a>
     </p>
 
+    <p>
+        We are introducing static membership towards KStream users. This feature reduces unnecessary rebalances during normal application upgrades or rolling bounces.
+        If you want to use static membership, first thing is to upgrade both broker cluster and stream apps to 2.3.0. This involves a necessity to change <code>inter.broker.protocol.version</code>
```

Comment:
`membership, [it is required] to upgrade your Kafka cluster (i.e., brokers) and your Kafka Streams application to 2.3.0.`

Reply:
Do we need to include `This involves a necessity to change <code>inter.broker.protocol.version</code> to 2.3 as well.` -- Maybe just link to the main upgrade section instead?

Reply:
I guess declaring the requirement here is important to make sure user won't choose to only upgrade one broker for coordinator as a shortcut.

Reply:
Yes, but this information is in the broker upgrade section already -- it seems better to not scatter information, but link to the corresponding upgrade sections? Or maybe I am missing your point?